docroot/modules/contrib/mirador/mirador_annotation/mirador_annotation.info.yml

```diff
@@ -11,3 +11,4 @@ core: '8.x'
 dependencies:
   - rest
   - mirador
+  - hal
```

Comment:
No..We are using application/hal+json format for annotation endpoint.
We need to enable hal_json format for methods. That's why added dependency to hal